### PR TITLE
Feature/fixbug on displaying feedback per ing name

### PR DESCRIPTION
**Pull request by Atos20**
#### What’s this PR do?  
- It fixes the bug on the displaying the proper names after the user has added the ingredients to the pantry 
- it fixes lint errors for all files
- it refactors the test files do the all are passing
- it removes unnecessary comments
- it links all the test files to dummy data files
- removes unnecessary global variables

#### Where should the reviewer start?  
 - `test files`
-  `script.js` 
- `domUpdates.js`

#### What are the relevant tickets?
closes #48 
#### How should this be manually tested?  
 run `npm test` on the terminal to run all the tet files 
`run npm` start on your terminal to run the localhost

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -141,19 +138,34 @@ class DomUpdates {
...
+        })
+        return element;
+      })
+      return editedMessage.join(' ');
+    });
+    newMessage.forEach(message => card.innerHTML += `
+      <ul class="feedback">
+        <li>${message}</li>
+      </ul>
+      `)
```
Comment (BlakeDonn) [inline src/domUpdates.js:164]: Looks good, I will try to make this so it doesn't say 'user' or 'unit', and instead says 'you have added 5 cloves of garlic'

**Review by BlakeDonn — APPROVED** [review]
Looks good, thanks Orlando

**Comment by BlakeDonn** [comment]
Going to go through and make a final sweep to remove any comments or anything not needed.

File: src/domUpdates.js (hunk 1 above)
Comment (Atos20) [reply src/domUpdates.js:164]: It was late when I finished this functionality but removing the user and replacing with `you`, should be ease, let me know if you want me to close the PR when you are done, thanks.

**Pull request merged**